src/System.Security.Cryptography.Csp/ref/System.Security.Cryptography.Csp.cs

```diff
@@ -7,6 +7,28 @@
 
 namespace System.Security.Cryptography
 {
+#if netcoreapp11
```

Comment:
Does this mean these types show up in the netcore contracts but not in the netstandard ones?